cdap-integration-test/src/main/java/io/cdap/cdap/test/remote/RemoteApplicationManager.java

```diff
@@ -192,8 +194,8 @@ public void delete() throws Exception {
   }
 
   @Override
-  public void upgrade() throws Exception {
-    applicationClient.upgradeApplication(application);
+  public void upgrade(@Nullable Set<String> artifactScopes, boolean allowSnapshot) throws Exception {
```

Comment:
Introduce new overload method instead of change method. Also use empty collection instead of `null`